docs/plugins/ingest-attachment.asciidoc

```diff
@@ -238,3 +238,58 @@ properties on this top level field will contain the value of the
 first attachment only. However, by specifying the
 `target_field` on to a value on `_ingest._value` it will correctly
 associate the properties with the correct attachment.
+
+[[ingest-attachment-raw-metadata]]
+==== Extracting all raw metadata from documents
+
+By default, the `ingest-attachment` plugin only extracts a subset of the most common metadata.
+
+If you want to get back all the raw metadata, you can set `properties` to `raw_metadata`.
+It will populate a subfield `raw_metadata` with all key/value pairs found as metadata in the document.
+
+For example:
+
+[source,js]
+--------------------------------------------------
+PUT _ingest/pipeline/attachment
+{
+  "description" : "Extract all metadata",
+  "processors" : [
+    {
+      "attachment" : {
+        "field" : "data",
+        "properties": [ "raw_metadata" ]
```

Comment:
Well. It's coming from https://github.com/dadoonet/fscrawler#disabling-raw-metadata where I'm actually putting all that stuff under a `meta.raw` field.